pg_replicate/src/clients/delta.rs

```diff
@@ -25,6 +33,87 @@ pub struct DeltaClient {
 }
 
 impl DeltaClient {
+    async fn aws_config() -> HashMap<String, String> {
+        let mut storage_options = HashMap::new();
+
+        storage_options.insert(AWS_FORCE_CREDENTIAL_LOAD.to_string(), "true".to_string());
+        storage_options.insert(AWS_S3_ALLOW_UNSAFE_RENAME.to_string(), "true".to_string());
+        let config = aws_config::load_defaults(BehaviorVersion::v2024_03_28()).await;
+
+        match config.endpoint_url() {
+            Some(endpoint) => {
+                storage_options.insert(AWS_ENDPOINT_URL.to_string(), endpoint.to_string());
+                storage_options.insert(AWS_ALLOW_HTTP.to_string(), "true".to_string());
```

Comment:
Why do we want to enable insecure non-TLS connections?

Reply:
this one if we want to test in minio or localstack